keyboards/tn_alice/info.json

```diff
@@ -0,0 +1,129 @@
+{
+    "manufacturer": "Tony Studio",
+    "keyboard_name": "tn_alice",
+    "maintainer": "thong-phn",
+    "bootloader": "uf2boot",
+    "board": "STM32_F103_STM32DUINO",

```

Comment:
```suggestion
```

Reply:
@zvecr Actually `"board": "STM32_F103_STM32DUINO"` is still needed here, otherwise `BOARD` defaults to `GENERIC_STM32_F103`, which does not actually exist.  Maybe the proper solution is to add a separate board for that bootloader (`platforms/chibios/boards/STM32_F103_STM32DUINO/board/board.c` does stm32duino-specific things with `BKP->DR10`), but currently all boards that use the `uf2boot` bootloader also specify `"board": "STM32_F103_STM32DUINO"` (sometimes indirectly through `"development_board": "bluepill"`).